includes/theme-support/class-llms-twenty-twenty-one.php

```diff
@@ -0,0 +1,333 @@
+		if ( 'editor' === $context ) {
+
+			// Elements with a light background that become unreadable in darkmode in the block editor.
+			$styles[] = LLMS_Theme_Support::get_css(
+				array(
+					'.llms-lesson-link .llms-lesson-title',
+					'.llms-lesson-link .llms-main > *',
+				),
+				array(
+					'color' => 'var( --global--color-background )',
+				),
+				'.is-dark-theme ' . $selector_prefix,
+			);
+		}
+
+		// Fix anchor buttons.
+		$styles[] = 'a.llms-button-action, a.llms-button-danger, a.llms-button-primary, a.llms-button-secondary { display: inline-block; }';
+
+		// Elements with a light background that become unreadable in darkmode.
+		$styles[] = LLMS_Theme_Support::get_css(
+			array(
+				'.is-dark-theme .llms-notification',
+				'.is-dark-theme .llms-table tbody tr:nth-child(odd) td',
+				'.is-dark-theme .llms-table tbody tr:nth-child(odd) td a',
+				'.is-dark-theme .llms-certificate-container',
+				'.is-dark-theme a.llms-certificate',
+				'.is-dark-theme .llms-instructor-info .llms-instructors',
+				'.is-dark-theme .llms-achievement-loop-item.achievement-item',
+				'.is-dark-theme .llms-achievement',
+				'.is-dark-theme .llms-loop-item-content .llms-loop-title:hover',
+				'.llms-notice a'
+			),
+			array(
+				'color' => 'var( --global--color-background )',
+			),
+			$selector_prefix
+		);
+		$styles[] = LLMS_Theme_Support::get_css(
+			array(
+				'.llms-checkout',
+				'.llms-access-plan .llms-access-plan-footer',
+				'.llms-access-plan .llms-access-plan-content',
+			),
+			array(
+				'background-color' => 'var( --global--color-background )',
+			),
+			$selector_prefix
+		);
+
+		// Add background color to qualifying elements.
+		$styles[] = LLMS_Theme_Support::get_css(
+			LLMS_Theme_Support::get_selectors_primary_color_background(),
+			array(
+				'color'            => 'var( --global--color-background )',
+				'background-color' => 'var( --global--color-secondary )',
+			),
+			$selector_prefix
+		);
+
+		// Add background color to qualifying elements.
+		$styles[] = LLMS_Theme_Support::get_css(
+			LLMS_Theme_Support::get_selectors_primary_color_border(),
+			array(
+				'border-color' => 'var( --global--color-secondary )',
+			),
+			$selector_prefix
+		);
+
+		// Add background color to qualifying elements.
+		$styles[] = LLMS_Theme_Support::get_css(
+			LLMS_Theme_Support::get_selectors_primary_color_text(),
+			array(
+				'color' => 'var( --global--color-secondary )',
+			),
+			$selector_prefix
+		);
+
+		return implode( "\r", $styles );
+
+	}
+
+	/**
+	 * Handle wrapping the catalog page header in 2021 theme elements.
+	 *
+	 * This method determines if the catalog title are to be displayed and adds additional actions
+	 * which will wrap the elements in 2021 theme elements depending on what is meant to be displayed.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public static function handle_page_header_wrappers() {
+
+		$show_title = apply_filters( 'lifterlms_show_page_title', true );
```

Comment:
We should document this filter somewhere, probably here: https://github.com/gocodebox/lifterlms/blob/4.9.0/templates/loop.php#L19 so that we can then say:
```suggestion
		/** This filter is documented in templates/loop.php */
		$show_title = apply_filters( 'lifterlms_show_page_title', true );
```